packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -21,51 +21,17 @@ const AssetSummary = () => {
         no_MF_account,
     } = traders_hub;
     const { is_logging_in, is_switching } = client;
-    const { getExchangeRate } = common;
```

Comment:
I can not see any usage of this `getExchangeRate` in new implementation, how we are apply this?

Reply:
@mahdiyeh-deriv function getTotalBalance() uses getExchangeRate function.